Documentation/devicetree/bindings/display/panel/panel-dsi.yaml

```diff
@@ -0,0 +1,114 @@
+# SPDX-License-Identifier: (GPL-2.0-only OR BSD-2-Clause)
+%YAML 1.2
+---
+$id: http://devicetree.org/schemas/display/panel/panel-dsi.yaml#
+$schema: http://devicetree.org/meta-schemas/core.yaml#
+
+title: Generic MIPI DSI Panel
+
+maintainers:
+  - Timon Skerutsch <timon@diodes-delight.com>
+
+allOf:
+  - $ref: panel-common.yaml#
+
+properties:
+  compatible:
+    description:
+      Shall contain a panel specific compatible and "panel-dsi"
+      in that order.
+    items:
+      - {}
+      - const: panel-dsi
+
+  dsi-color-format:
+    description: |
+      The color format used by the panel. Only DSI supported formats are allowed.
+    enum:
+      - RGB888
+      - RGB666
+      - RGB666_PACKED
+      - RGB565
+
+  lanes:
+    description: |
+      The number of lanes used by the panel.
+    $ref: /schemas/types.yaml#/definitions/uint32
+    enum: [1, 2, 3, 4]
```

Comment:
The more consistent approach would be to use data-lanes as an array of data lane indexes. This is used across a number of bridges, and is consistent with V4L2.
eg https://github.com/torvalds/linux/blob/master/Documentation/devicetree/bindings/display/bridge/chipone,icn6211.yaml#L64-L71

Reply:
I'll admit that DRM is less consistent than V4L2, but some devices support reordering of the data lanes. Using the array allows for a consistent way of configuring the interface.

Any non-standard property is meant to be prefixed with a vendor code. `data-lanes` is standardised. `lanes` is not. Friendly advice if upstreaming - use the standard properties.

Reply:
Thats fair yea. I will change it to `data-lanes`.